.circleci/config.yml

```diff
@@ -361,15 +367,13 @@ commands:
       - run:
           name: Build torch-tensorrt python release with only the fx backend
           command: |
-            export CUDA_HOME=/usr/local/cuda-12.1/
+            export CUDA_HOME=/usr/local/cuda-<< parameters.cuda-version >>/
             mv toolchains/ci_workspaces/WORKSPACE.<< parameters.platform >> WORKSPACE
-            cd py
-            python3 -m pip install wheel setuptools
-            python3 -m pip install pybind11==2.6.2
-            python3 setup.py bdist_wheel --fx-only
-            python3 setup.py install --fx-only
+            python3 -m pip install pip==22.0.2
```

Comment:
We need to follow up with TensorRT that their python package fails to build with pip versions > 22.x cc @Christina-Young-NVIDIA, @laikhtewari